k8s/namespaces/admin/traefik-private/patches/demo-01.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: admin
 spec:
   values:
-    loadBalancerIP: 10.51.31.251
\ No newline at end of file
+    loadBalancerIP: 10.50.80.251
```

Comment:
Can you review these IPs? they don't look correct compared to last 10.50.XXX.0 of each subnet